tests/end_to_end/candid_rpc/class_syntax/call_raw/src/index.ts

```diff
@@ -8,7 +8,7 @@ export default class {
         candidArgs: string,
         payment: bigint
     ): Promise<string> {
-        const result = await call(canisterId, method, {
+        const result = await call<undefined>(canisterId, method, {
```

Comment:
Why is this undefined? I'm sure there is a good reason, it just looks out of the ordinary so I thought I would ask